client/MentorPanel/src/components/questions.js

```diff
@@ -20,22 +19,20 @@ const Questions = () => {
 
   const questions = mentor.topic_questions[current_topic] || []
   const recommended = mentor.topic_questions['Recommended'] || []
-  const height = document.getElementById('question-container').clientHeight
+  const height = (window.innerHeight * 0.5) - 120
```

Comment:
can we have the height for these components passed in a prop? Seems like what we want is to make these components as reusable as possible and the decision for height should be long to a parent component that owns the layout